testing/tests/DevExpress.ui.widgets/tooltip.tests.js

```diff
@@ -37,6 +41,46 @@ QUnit.test("render as tooltip", function(assert) {
     assert.ok(wrapper().length);
 });
 
+QUnit.test("tooltip should render when target is core renderer object", function(assert) {
```

Comment:
For new tests use ES6, pls.

// Remember to replace "require" with "import" at the beginning of the file